storybook/src/docs/components/DataGrid/DataGridExcelExport.stories.tsx

```diff
@@ -0,0 +1,213 @@
+import { gql, useQuery } from "@apollo/client";
+import {
+    Button,
+    FileIcon,
+    FillSpace,
+    GridColDef,
+    Loading,
+    Toolbar,
+    ToolbarActions,
+    useBufferedRowCount,
+    useDataGridExcelExport,
+    useDataGridRemote,
+} from "@comet/admin";
+import { MoreVertical } from "@comet/admin-icons";
+import { Menu, MenuItem } from "@mui/material";
+import Box from "@mui/material/Box";
+import { DataGrid } from "@mui/x-data-grid";
+import type { Meta, StoryFn } from "@storybook/react";
+import { useRef, useState } from "react";
+
+import { apolloStoryDecorator } from "../../../apollo-story.decorator";
+import { storyRouterDecorator } from "../../../story-router.decorator";
+
+type Story = StoryFn;
+const config: Meta = {
+    title: "Docs/Components/DataGrid/DataGridExcelExport",
+    decorators: [storyRouterDecorator(), apolloStoryDecorator("/graphql")],
+};
+export default config;
+
+type Launch = {
+    id: string;
+    mission_name: string;
+};
+
+interface LaunchesPastResultData {
+    data: Launch[];
+    result: { totalCount: number };
+}
+
+interface GQLQueryVariables {
+    offset: number;
+    limit: number;
+    sort?: string;
+    order?: string | null;
+}
+
+interface GQLQuery {
+    __typename?: "Query";
+    launchesPastResult: LaunchesPastResultData;
+}
+
+type ExcelLaunch = {
+    id: string;
+    mission_name: string;
+    launch_date_local: Date;
+};
+
+interface ExcelLaunchesPastResultData {
+    data: ExcelLaunch[];
+    result: { totalCount: number };
+}
+
+interface GQLExcelQueryVariables {
+    offset: number;
+    limit: number;
+    sort?: string;
+    order?: string | null;
+}
+
+interface GQLExcelQuery {
+    __typename?: "Query";
+    launchesPastResult: ExcelLaunchesPastResultData;
+}
```

Comment:
It's cumbersome that we need to type this for every story. Maybe we could move the base types to a separate file and use `Pick` to achieve the correct types?